library/pod-security-policy/users/example.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis-replica
+spec:
+  selector:
+    matchLabels:
+      app: redis
+      role: replica
+      tier: backend
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: redis
+        role: replica
+        tier: backend
+    spec:
+      containers:
+        - name: replica
+          args: ["--port=8080"]
+          image: k8s.gcr.io/test-webserver
+          resources:
+            requests:
+              cpu: 100m
+              memory: 100Mi
+          ports:
+            - containerPort: 6379
+          securityContext:
+            runAsUser: 177 # constraint range is 100-200
```

Comment:
All the other examples use negative cases to ensure the deployed example + constraints + constraint template will end in a rejection so users can see how this policy works. can we use a number outside of the constraint range so that this example actually fails to create?

Reply:
Done